docs/images/logo_wide_v1.svg

```diff
@@ -0,0 +1 @@
et="0" stop-color="#2c2fa9"/><stop offset="1" stop-color="#504cff"/></linearGradient><g stroke-width="1.133623"><path d="m276.4441 7.7520268v53.2802592h-26.64013c-34.80222 0-63.6189 22.241674-74.59236 53.280254h-63.93632c-10.97347-31.03858-39.790153-53.280254-74.592365-53.280254h-26.64013v-53.2802592h26.64013a133.00793 133.00793 0 0 1 106.560525 53.2802592 133.00793 133.00793 0 0 1 106.56052-53.2802592z" fill="url(#d)"/><path d="m249.80397 220.87306h26.64013v53.28026h-26.64013a133.00793 133.00793 0 0 1 -106.56052-53.28026 133.00793 133.00793 0 0 1 -106.560525 53.28026h-26.64013v-53.28026h26.64013c34.802212 0 63.618895-22.24167 74.592365-53.28026h63.93632c10.97346 31.03859 39.79014 53.28026 74.59236 53.28026z" fill="url(#c)"/><path d="m175.21158 114.31253h101.2325v53.280262h-101.2325z" fill="url(#b)"/><path d="m10.042817 114.31253h101.2325v53.280262h-101.2325z" fill="url(#a)"/><g fill="#090c36"><path d="m383.00456 220.87306v-159.840774h62.34924q37.21682 0 61.07958 22.672451 23.86275 22.672443 23.80607 56.749133v.45346q0 34.24674-23.80607 57.08923-23.80607 22.84249-61.07958 22.8765zm35.1423-31.74143h27.20694q21.45948 0 34.82488-13.24071 13.36541-13.24071 13.35408-34.4848v-.45345q0-21.24409-13.35408-34.71152-13.35407-13.467431-34.82488-13.467431h-27.20694z"/><path d="m629.05732 223.54841q-35.14229 0-58.51759-23.72672-23.3753-23.72672-23.38663-58.41557v-.45345q0-34.66617 23.49999-58.642289 23.5-23.976116 59.77593-23.953444 21.22141 0 36.04919 6.280269a84.443543 84.443543 0 0 1 26.92354 18.364685l-22.35504 25.778569q-19.86107-18.024591-40.81041-18.024591a42.941622 42.941622 0 0 0 -33.11311 14.487701q-13.24071 14.48769-13.24071 35.25565v.45345q0 20.9947 13.12734 35.59575 13.12735 14.60106 33.19247 14.60106 12.77593 0 22.12831-4.5345a86.586089 86.586089 0 0 0 19.84973-14.13627l22.36637 22.58177a94.555456 94.555456 0 0 1 -28.40858 21.27809q-15.16787 7.22117-37.0808 7.20984z"/><path d="m718.62485 220.80504v-159.704736h73.00528q30.34708 0 46.54654 16.199466 13.69417 13.69416 13.69417 36.50265v.45344q0 36.75204-34.22407 49.51663l39.00795 57.03255h-41.0598l-34.22407-51.1037h-27.61504v51.1037zm35.1423-82.13095h35.59574q12.54921 0 19.73637-6.16691 7.18717-6.1669 7.18717-16.42619v-.45345q0-11.16618-7.30053-17.004334-7.30053-5.838157-20.31452-5.849487h-34.91557z"/><path d="m879.53123 220.80504v-159.704736h62.28122q37.18282 0 61.03425 22.67245 23.8513 22.672446 23.806 56.681116v.45346q0 34.22406-23.806 57.04389-23.80608 22.81982-61.03425 22.80848zm35.14229-31.74143h27.15027q21.44814 0 34.80221-13.2407 13.35407-13.24072 13.34274-34.4508v-.45344q0-21.22143-13.34274-34.67752-13.34274-13.456101-34.84755-13.467431h-27.10493z"/><path d="m1047.0466 220.80504v-159.704736h120.4247v31.253972h-85.5545v32.398934h75.3519v31.25396h-75.2952v33.5439h86.6995v31.25397z"/><path d="m1188.0466 220.80504 55.6722-81.22405-53.3937-78.480686h41.0712l33.5665 51.783876 33.7707-51.783876h39.9148l-53.3936 78.027236 55.6608 81.6775h-41.0711l-35.8225-54.75396-36.0492 54.75396z"/></g></g></svg>
```

Comment:
I'm not crazy about creating an entire two-deep top-level directory for a single image. I'm taking to suggestions on other places to put this.

Reply:
Doesn't bother me.